src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/QuestionManager.java

```diff
@@ -118,7 +118,7 @@ public final Response validateAnswer(final Question question, final ChoiceDTO su
      * @return a Validator
      */
     @SuppressWarnings("unchecked")
-    private static IValidator locateValidator(final Class<? extends Question> questionType) {
+    public static IValidator locateValidator(final Class<? extends Question> questionType) {
```

Comment:
I think @sacummins should pronounce on this, since it changes the encapsulation of functionality.

Reply:
I guess this has become necessary because the facade layer is doing a bit more than I had intended it to do. 

The logic for determining whether a testCase works or not should probably be in the manager layer. 

Originally, the facade layer was intended to be quite lightweight and was just there to do things like authentication / authorisation, user input validation etc. Overtime more stuff seems to have crept in the facade. The aim of the separation (manager layer) was to allow separation of the business logic from the exposed API bits.

Reply:
This is a good observation. I think I'll be able to move things around to better fit the existing architecture.